Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -33,15 +33,20 @@ private void SyncAtlassianWorklogs(long instanceId)
 
             SyncDeletedWorklogs(instanceId, lastSync);
             SyncUpdatedWorklogs(instanceId, lastSync);
+
+            UpdateLastSyncDate(instanceId);
         }
 
         private DateTime GetLastSyncDate(long instanceId)
         {
             using (var db = new ReportsDb())
             {
-                var worklogs = db.AtlassianWorklogs.OrderByDescending(p => p.LastSync).FirstOrDefault();
+                var lastSync = db.InstalledInstances
+                    .Where(p => p.Id == instanceId)
```

Comment:
Use First/Single instead of Where, because we do not expect a list, we expect a single instance returned.